### PR TITLE
update changelog and version for v1.4.4

### PR DESCRIPTION
**Pull request by liamcervante**

### CODE REVIEW

**Review by dekimsey — APPROVED** [review]

**Review by radditude — APPROVED** [review]

File: version/version.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // The main version number that is being run at the moment.
-var Version = "1.4.3"
+var Version = "1.4.4"
```
Comment (alisdair) [inline version/version.go:14]: I might be misreading but I thought we should update _only_ version/VERSION and leave this for the automation to increment.

Comment (liamcervante) [reply version/version.go]: yep, reverted!

Comment (maxb) [reply version/version.go]: No automation updated the versions in version.go, before the Git tag was made... was it actually correct to not update this?

Comment (liamcervante) [reply version/version.go]: Yes we should have updated this. I've updated our internal release documentation to make this clear.

**Pull request merged**